diff --git a/cmake/draco_options.cmake b/cmake/draco_options.cmake
index fa460c6..79af7f2 100644
--- a/cmake/draco_options.cmake
+++ b/cmake/draco_options.cmake
@@ -219,7 +219,7 @@ macro(draco_set_optional_features)
     endif()
 
 
-    if(NOT EMSCRIPTEN)
+    if(TRUE) # if(NOT EMSCRIPTEN)
       # For now, enable deduplication for both encoder and decoder.
       # TODO(ostava): Support for disabling attribute deduplication for the C++
       # decoder is planned in future releases.
