diff --git a/src/draco/io/gltf_decoder.cc b/src/draco/io/gltf_decoder.cc
index 2ea71ec..eac0002 100644
--- a/src/draco/io/gltf_decoder.cc
+++ b/src/draco/io/gltf_decoder.cc
@@ -497,7 +497,7 @@ Status GltfDecoder::LoadFile(const std::string &file_name,
       // TinyGLTF's ExpandFilePath does not do filesystem i/o, so it's safe to
       // use in all environments.
       &tinygltf::ExpandFilePath, &ReadWholeFile, &WriteWholeFile,
-      reinterpret_cast<void *>(input_files)};
+      [](unsigned long*, std::string*, const std::string&, void*){ return true; }};
 
   loader.SetFsCallbacks(fs_callbacks);
 
